### PR TITLE
context fix string output + release 0.4.1

### PR DESCRIPTION
**Pull request by sanderland**

### CODE REVIEW

File: cognite/experimental/data_classes/contextualization.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import copy
 import math
+from typing import *
 from typing import Iterable, List, Union
```
Comment (andreavs) [inline cognite/experimental/data_classes/contextualization.py]: probably remove this import then

File: cognite/experimental/data_classes/relationships.py [hunk 2]
```diff
@@ -93,10 +93,7 @@ class RelationshipFilter(CogniteFilter):
         confidence (Dict[str, Any]): Range to filter the field for. (inclusive)
         last_updated_time (Dict[str, Any]): Range to filter the field for. (inclusive)
         created_time (Dict[str, Any]): Range to filter the field for. (inclusive)
-        active_at_time (int): Limits results to those active at this time, i.e. activeAtTime falls between startTime and
-            endTime startTime is treated as inclusive (if activeAtTime is equal to startTime then the relationship will be included).
-            endTime is treated as exclusive (if activeTime is equal to endTime then the relationsip will NOT be included).
-            If a relationship has neither startTime nor endTime, the relationship is active at all times
+        active_at_time (int): Limits results to those active at this time, i.e. activeAtTime falls between startTime and endTime startTime is treated as inclusive (if activeAtTime is equal to startTime then the relationship will be included). endTime is treated as exclusive (if activeTime is equal to endTime then the relationsip will NOT be included). If a relationship has neither startTime nor endTime, the relationship is active at all times
```
Comment (andreavs) [inline cognite/experimental/data_classes/relationships.py]: is this autoformatting or do you prefer it this way?

Comment (sanderland) [reply cognite/experimental/data_classes/relationships.py]: data classes are autogenerated

File: cognite/experimental/data_classes/contextualization.py (hunk 1 above)
Comment (sanderland) [reply cognite/experimental/data_classes/contextualization.py]: data classes are autogenerated

Comment (andreavs) [reply cognite/experimental/data_classes/contextualization.py]: wait it seriously autogenerates "import *" followed by "import thing"

Comment (sanderland) [reply cognite/experimental/data_classes/contextualization.py]: no it generates from typing import * if it doesn't exist in the files, so the other thing is redundant

**Pull request merged**